mama/c_cpp/src/c/subscription.c

```diff
@@ -1194,9 +1202,11 @@ mamaSubscription_getSubjectContext (mamaSubscription subscription,
         msgUtils_getIssueSymbol (msg, &issueSymbol);
         context->mSymbol = copyString (issueSymbol);
         #ifdef WITH_ENTITLEMENTS
+		{
```

Comment:
More WITH_ENTITLEMENTS  stuff - please remove along with unnecessary whitespace change at the end of this function.